CHANGES.rst

```diff
@@ -46,6 +46,8 @@ Released on XXX
 
 * **Drop support of charade, now that chardet is supported once more.**
 
+* Added support for self closing ``<col />`` tags.
```

Comment:
In general, we keep small changes out of the changes file. (Does the contributing doc say that? If not, bah! That really should've been caught before now…)